tests/test_serializer_nested.py

```diff
@@ -38,3 +38,88 @@ def test_nested_serialize_empty(self):
         }
         serializer = self.Serializer()
         assert serializer.data == expected_data
+
+
+class TestNestedSerializerWithMany:
+    def setup(self):
+        class PaymentInfoSerializer(serializers.Serializer):
+            url = serializers.URLField()
+            amount = serializers.DecimalField(max_digits=6, decimal_places=2)
+
+        class NestedSerializer(serializers.Serializer):
+            foo = serializers.CharField()
```

Comment:
Could we just have a single `NestedSerializer` with a field named `example`, and use the same nested serializer for both the `allow_null` and `not_allow_null` cases?